releasenotes/notes/improve-systemd-config-a82bca0014f6ccc6.yaml

```diff
@@ -0,0 +1,20 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+upgrade:
+  - |
+    Update systemd configuration. Before if `unit_names` and `unit_regexes`
+    are not set, all units are monitored. Now `unit_names` is required and only
+    matching units will be monitored, `unit_regexes` have been removed (avoid
+    accidental monitoring of too many metrics).
+  - |
+    New systemd configuration `private_socket`, and `max_units` have been added.
+    Default is now prefixed with `/host` when using Docker Agent.
```

Comment:
this is not an `upgrade` note since it's not a change that requires action from users when they upgrade. This should be under `features` or `enhancements` if you want to mention them. I don't even know if we should really mention this since the integration is not publicly released at the moment.

Reply:
I updated this section with some improvement (+ removed `max_units`, it was a typo since it have been removed from implementation)

I think it could help if we keep this note since there is a change in the default when using Docker Agent.